### ISSUE TITLE
How this thing works? 

### ISSUE THREAD

**Issue by ill02** [winstxnhdw/Auto-IdleRPG#1]
i cant find any spot to put my token or sum

**Comment by winstxnhdw** [comment]
Have you run the script?

**Comment by ill02** [comment]
i installed the .bat then run the .js script

**Comment by winstxnhdw** [comment]
> i installed the .bat then run the .js script

1. Have you installed Python?
2. It should be a .py script not .js

**Comment by ill02** [comment]
> 
>     1. Have you installed Python?
> 
>     2. It should be a .py script not .js

1. yes i got python 3.9 installed
2. my bad, just a typo

**Comment by winstxnhdw** [comment]
> > 
> 
> >     1. Have you installed Python?
> 
> > 
> 
> >     2. It should be a .py script not .js
> 
> 
> 
> 1. yes i got python 3.9 installed
> 
> 2. my bad, just a typo 
> 
> 
> 
> 

Okay. What happens when you run the script? Does it say anything?

**Comment by ill02** [comment]
it does nothing

**Comment by winstxnhdw** [comment]
> it does nothing 

How are you running the script. Through the command prompt? Make sure to change your terminal directory to the directory where you left the script.

**Comment by ill02** [comment]
>How are you running the script?Through the command prompt?

yes ive already tried it on command prompt also in direct in python.

>Make sure to change your terminal directory to the directory where you left the script.

ik, its just really dont function.

**Comment by winstxnhdw** [comment]
@ill02 Hey, I tried running the script on my computer again, it works just fine. Are you sure there is no output from the terminal?
